src/marketing/price-rule-catalog-scheduled-changes.md

```diff
@@ -3,12 +3,87 @@ conditions: Default.EE-B2B
 title: Scheduled Changes for Catalog Price Rules
 ---
 
-The Scheduled Changes box appears at the top of the page when a new price rule is saved for the first time. Catalog price rules can be applied on schedule as part of a campaign, and grouped with other content changes. You can create a new campaign based on scheduled changes to a price rule, or apply the changes to an existing campaign.
+The Scheduled Changes box appears at the top of the page when a new price rule is saved or updated. Catalog price rules can be applied on schedule as part of a campaign, and grouped with other content changes. You can create a new campaign based on scheduled changes to a price rule, or apply the changes to an existing campaign.
 
 If there are multiple price rules running in the same campaign, the Priority setting of the price rule determines which rule takes precedence. To learn more, see [Content Staging]({% link cms/content-staging.md %}).
 
 {: .bs-callout .bs-callout-info}
 If a campaign that includes a price rule is initially created without an end date, the campaign cannot be edited later to include an end date. It is recommended that you either add an end date when you create the campaign, or create a duplicate version of the existing campaign and add the end date to the duplicate as needed.
 
-![Catalog price rule - scheduled changes]({% link images/images-ee/content-staging-price-rule-catalog-scheduled-change.png %}){: .zoom}
+![Catalog price rule - scheduled changes]({% link images/images-ee/price-rule-catalog-scheduled.png %}){: .zoom}
 _Scheduled Changes_
+
+## Schedule an update to a catalog price rule
+
+1. On the _Admin_ sidebar, go to **Marketing** > _Promotions_ > **Catalog Price Rule**.
+
+1. Open the rule in edit mode.
+
+1. In the **Scheduled Changes** box at the top of the page, click <span class="btn">Schedule New Update</span>.
+
+1. With the **Save as a New Update** option selected, do the following:
+
+    - In the **Update Name** field, enter a name for the update to the rule.
+
+    - Enter a brief **Description** of the update, including how or why it is applied.
+
+    - Use the Calendar (![]({% link images/images/btn-calendar.png %}){: .Inline}) to choose the **Start Date** and **End Date** for the scheduled change to be in effect. To create an open-ended change, leave the End Date blank.
+
+    {:.bs-callout .bs-callout-info}
```

Comment:
Does merchdocs still use this construction for note blocks? In devdocs, it's just `{:.bs-callout-info}`